src/write.ts

```diff
@@ -200,32 +206,77 @@ export function buildComment(
+        const line = `| \`${current.name}\` | ${strVal(current)} | ${strVal(prev)} | \`${floatStr(ratio)}\` |`;
+        output.push(line);
+    }
+}
+
+const RESULT_TABLE_HEADER = ['', `| Benchmark suite | Current | Previous | Ratio |`, '|-|-|-|-|'];
+
+function buildReportComment(
+    results: Alert[],
+    benchName: string,
+    curSuite: Benchmark,
+    prevSuite: Benchmark,
+    cc: string[],
+): string {
+    // Do not show benchmark name if it is the default value 'Benchmark'.
+    const benchmarkText = benchName === 'Benchmark' ? '' : ` **'${benchName}'**`;
+    const lines = [
+        '# Performance Report',
+        '',
+        `For benchmark${benchmarkText}.`,
+        '',
+        `Previous commit: ${prevSuite.commit.id}`,
+        `Current commit: ${curSuite.commit.id}`,
+    ];
+
+    lines.push(...RESULT_TABLE_HEADER);
+    pushResultLines(results, lines);
+    lines.push('', commentFooter());
+
+    if (cc.length > 0) {
+        lines.push('', `CC: ${cc.join(' ')}`);
+    }
+
+    return lines.join('\n');
+}
+
 function buildAlertComment(
-    alerts: Alert[],
+    losses: Alert[],
+    gains: Alert[],
     benchName: string,
     curSuite: Benchmark,
     prevSuite: Benchmark,
     threshold: number,
     cc: string[],
 ): string {
     // Do not show benchmark name if it is the default value 'Benchmark'.
-    const benchmarkText = benchName === 'Benchmark' ? '' : ` **'${benchName}'**`;
-    const title = threshold === 0 ? '# Performance Report' : '# :warning: **Performance Alert** :warning:';
+    const benchmarkText = benchName === 'Benchmark' ? '' : ` for **'${benchName}'**`;
     const thresholdString = floatStr(threshold);
     const lines = [
-        title,
+        `# Performance Report${benchmarkText}`,
         '',
-        `Possible performance regression was detected for benchmark${benchmarkText}.`,
-        `Benchmark result of this commit is worse than the previous benchmark result exceeding threshold \`${thresholdString}\`.`,
+        `Benchmark result(s) exceed ratio of \`${thresholdString}\`.`,
         '',
-        `| Benchmark suite | Current: ${curSuite.commit.id} | Previous: ${prevSuite.commit.id} | Ratio |`,
-        '|-|-|-|-|',
+        `Previous commit: ${prevSuite.commit.id}`,
+        `Current commit: ${curSuite.commit.id}`,
     ];
 
-    for (const alert of alerts) {
-        const { current, prev, ratio } = alert;
-        const line = `| \`${current.name}\` | ${strVal(current)} | ${strVal(prev)} | \`${floatStr(ratio)}\` |`;
-        lines.push(line);
+    if (gains.length > 0) {
+        lines.push(...['', '### :rocket: The following benchmarks show improvements:']);
+        lines.push(...RESULT_TABLE_HEADER);
+        pushResultLines(gains, lines);
+    }
+
+    if (losses.length > 0) {
+        lines.push(...['', '### :snail: The following benchmarks show regressions:']);
+        lines.push(...RESULT_TABLE_HEADER);
+        pushResultLines(losses, lines);
     }
```

Comment:
Let's put performance losses as the first one in case we have both at the same time. I believe those are more important to see at first glance

Reply:
:+1: Done